CONVENTIONS.md

```diff
@@ -63,6 +63,7 @@ But the runtime would fill in the mappings so the plugin itself would receive so
 | dns | Dynamically configure dns according to runtime | `dns` | Dictionary containing a list of `servers` (string entries), a list of `searches` (string entries), a list of `options` (string entries). <pre>{ <br> "searches" : [ "internal.yoyodyne.net", "corp.tyrell.net" ] <br> "servers": [ "8.8.8.8", "10.0.0.10" ] <br />} </pre> | kubernetes | CNI `win-bridge` plugin, CNI `win-overlay` plugin |
 | ips | Dynamically allocate IPs for container interface. Runtime which has the ability of address allocation can pass these to plugins.  | `ips` | A list of `IP` (string entries). <pre> [ "10.10.0.1/24", "3ffe:ffff:0:01ff::1/64" ] </pre> | none | CNI `static` plugin |
 | mac | Dynamically assign MAC. Runtime can pass this to plugins which need MAC as input. | `mac` | `MAC` (string entry). <pre> "c2:11:22:33:44:55" </pre> | none | CNI `tuning` plugin |
+| guid | Dynamically assign GUID to network interface. Runtime can pass this to plugins which need GUID as input. | `guid` | `GUID` (string entry). <pre> "c2:11:22:33:44:55:66:77" </pre> | none | CNI `ib-sriov-cni` plugin |
```

Comment:
two nits:
- Runtime implementations of guid should not be `none`, maybe `multus`?
- Plugin Implementations of guid is better to be a link because it's a 3rd party plugin

Reply:
-  Yes the plan is to support it it in multus I will update the PR
- Yes I will update with the link to the ib-sriov-cni  in Mellanox github

Reply:
IMO calling Multus as runtime is not a good direction. It's multiplexer which is called by "real" runtime, so IMO `none` is there more correct.
+1 for url

Reply:
Ok, so for now I just update the URL for the ib-sriov-cni